templates/status/404.tmpl

```diff
@@ -1,8 +1,8 @@
 {{template "base/head" .}}
-<div role="main" aria-label="{{.Title}}" class="page-content ui container center gt-w-screen {{if .IsRepo}}repository{{end}}">
+<div role="main" aria-label="{{.Title}}" class="page-content ui center gt-w-screen {{if .IsRepo}}repository{{end}}">
 	{{if .IsRepo}}{{template "repo/header" .}}{{end}}
 	<div class="ui container center">
-		<p style="margin-top: 100px"><img src="{{AssetUrlPrefix}}/img/404.png" alt="404"></p>
+		<p style="margin-top: 100px"><img src="{{AssetUrlPrefix}}/img/404.png" alt="404" style="max-width: 100%"></p>
```

Comment:
It seems that there are already some good-enough helpers for this case.

Maybe it should be: `<img class="gt-mt-5 gt-w-100" src="{{AssetUrlPrefix}}/img/404.png" alt="404">`

(Although "avoiding inline styles as much as possible" hasn't been written into guideline yet, it has been a consensus for long time during reviewing.